mod/01_rating/set/type_plus_right/metric_value/value.rb

```diff
@@ -30,9 +30,37 @@ def metric_plus_company_card
   Card.fetch metric_plus_company
 end
 
+def unknown_value?
+  content.casecmp('unknown') == 0
+end
+
+def option_names metric_name
+  # value options
+  option_card = Card.fetch "#{metric_name}+value options", new: {}
+  option_card.item_names context: :raw
+end
+
 format :html do
+  view :select do |args|
+    options = [['-- Select --', '']] +
+              card.option_names(args[:metric_name]).map { |x| [x, x] }
+    select_tag('card[subcards][+values][content]',
+               options_for_select(options),
+               class: 'pointer-select form-control'
+              )
+  end
+
+  view :editor do |args|
```

Comment:
nice.  you overrode :editor (as opposed to :new, as you said in PT).  This is the better way.